artemis-core-client/src/main/java/org/apache/activemq/artemis/core/protocol/core/impl/ActiveMQClientProtocolManager.java

```diff
@@ -499,8 +500,7 @@ public void handlePacket(final Packet packet) {
 
       private void handleDisconnect(SimpleString nodeID, DisconnectReason reason, SimpleString targetNodeID, TransportConfiguration tagetConnector) {
          if (topologyResponseHandler != null) {
-            topologyResponseHandler.nodeDisconnected(conn, nodeID == null ? null : nodeID.toString(), reason,
-                    targetNodeID == null ? null : targetNodeID.toString(), tagetConnector);
+            topologyResponseHandler.nodeDisconnected(conn, Objects.toString(nodeID), reason, Objects.toString(targetNodeID), tagetConnector);
```

Comment:
Since `Object.toString(x)` returns "null" and not `null` in the case the value is `null` I'm worried there might be some behavioral changes here that are not intended.  In this case if `nodeID` is null the method will be called with `nodeID` set to "null" which might not be what was intended, I'd guess other cases in this PR might see similar side effects.

Reply:
I'll use `Objects.toString(String, null)` to keep the same semantics.